src/java.base/share/classes/jdk/internal/ref/CleanerImpl.java

```diff
@@ -225,7 +225,7 @@ public Thread newThread(Runnable r) {
         CleanerCleanable(Cleaner cleaner) {
             super(cleaner, cleaner);
         }
-
+java
```

Comment:
Stray change.

Reply:
Not sure what the "java" is for here seems like a typo

Reply:
Yes, sorry.